.gitignore

```diff
@@ -99,3 +99,4 @@ ENV/
 
 # mypy
 .mypy_cache/
+Rethinking_2/End_of_chapter_problems/Chapter_2_mine.ipynb
```

Comment:
Create a personal gitignore for this 

https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

Reply:
Fixed